src/dev.docker-compose.yml.jinja

```diff
@@ -96,6 +96,8 @@ services:
       - ./data/sessions:/data/odoo/sessions
       # mount home dir
       # - ./.home/.bash_history:/home/odoo/.bash_history
+      # mount backup directory
+      - ./backup:/odoo/backup
```

Comment:
Should we create a new dir at the root of the project or create a generic dir at the root and add backup as subfolder ? @Kev-Roche  @hparfr